lib/ansible/utils/display.py

```diff
@@ -448,7 +459,14 @@ def verbose(self, msg, host=None, caplevel=2):
             else:
                 self.display("<%s> %s" % (host, msg), color=C.COLOR_VERBOSE, stderr=to_stderr)
 
-    def get_deprecation_message(self, msg, version=None, removed=False, date=None, collection_name=None):
+    def get_deprecation_message(
+        self,
+        msg: str,
+        version: t.Any = None,
```

Comment:
Try to avoid `Any` at any cost. Also, this lacks an explicit `None` option.
Plus, looking at the use, it seems to be a string:
```suggestion
        version: str | None = None,
```